clint.py

```diff
@@ -1207,7 +1207,6 @@ def CheckForMultilineCommentsAndStrings(filename, clean_lines, linenum, error):
     ('getpwuid(', 'getpwuid_r('),
     ('gmtime(', 'gmtime_r('),
     ('localtime(', 'localtime_r('),
-    ('rand(', 'rand_r('),
```

Comment:
OTOH, we need something to replace that one `rand_r` call in `sha256.c` at some point. But should it be `os_rand` or `os_rand_r`?